src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/message/MergeService.kt

```diff
@@ -19,27 +20,28 @@ import uk.gov.justice.digital.hmpps.personrecord.service.TelemetryService
 import uk.gov.justice.digital.hmpps.personrecord.service.type.TelemetryEventType.CPR_MERGE_RECORD_NOT_FOUND
 import uk.gov.justice.digital.hmpps.personrecord.service.type.TelemetryEventType.CPR_RECORD_MERGED
 
-@Service
+@Component
```

Comment:
@Component and @Service do the same thing anyway. Shall we just go for @Component throughout?